### DISCUSSION TITLE
String in source code

### DISCUSSION THREAD

**Discussion by freelovehere** [postalserver/postal#2966] [Help with using Postal]
Can you please point me to the direction to locate the FROM which I circled in the image below, thank you . 
![IMG_1423](https://github.com/postalserver/postal/assets/169105514/c2bea375-7aa7-4ab3-8401-6dd8410827a0)

**Comment by willpower232** [comment]
https://github.com/postalserver/postal/blob/d5104991903d42563c6c6f7ff01276d496e3304f/app/controllers/messages_controller.rb#L10-L24
